docs/rest.md

```diff
@@ -310,22 +305,16 @@ If present, only list snaps that match the query.
 
 ### POST
 
-* Description: Sideload a snap to the system.
+* Description: Install an uploaded snap to the system.
```

Comment:
"uploaded" here isn't really clear. Maybe something like "install a snap provided in the request body..."?

No, that's not much better. Hmm.